modules/serverless-install-kafka-odc.adoc

```diff
@@ -0,0 +1,49 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-install-kafka-odc_{context}"]
+= Installing Apache Kafka components using the web console
+
+Cluster administrators can enable the use of Apache Kafka functionality in an {ServerlessProductName} deployment by instantiating the `KnativeKafka` custom resource definition provided by the *Knative Kafka* {ServerlessOperatorName} API.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Eventing are installed.
+* You have access to a Red Hat AMQ Streams cluster.
+* You have cluster administrator permissions on {product-name}.
+* You are logged in to the web console.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Operators* → *Installed Operators*.
+. Check that the *Project* dropdown at the top of the page is set to *Project: knative-eventing*.
+. Click *Knative Kafka* in the list of *Provided APIs* for the {ServerlessOperatorName} to go to the *Knative Kafka* tab.
+. Click *Create Knative Kafka*.
+. Optional: Configure the *KnativeKafka* object in the *Create Knative Kafka* page. To do so, use either the default form provided or edit the YAML.
+.. Using the form is recommended for simpler configurations that do not require full control of *KnativeKafka* object creation.
+.. Editing the YAML is recommended for more complex configurations that require full control of *KnativeKafka* object creation. You can access the YAML by clicking the *Edit YAML* link in the top right of the *Create Knative Kafka* page.
+. Click *Create* after you have completed any of the optional configurations for Kafka. You are automatically directed to the *Knative Kafka* tab where *knative-kafka* is in the list of resources.
+
+.Verification steps
+
+. Click on the *knative-kafka* resource in the *Knative Kafka* tab. You will be automatically directed to the *Knative Kafka Overview* page.
+. View the list of *Conditions* for the resource and confirm that they have a status of *True*.
++
+image::knative-kafka-overview.png[Kafka Knative Overview page showing Conditions]
++
+If the conditions have a status of `Unknown` or `False`, wait a few moments and then try again.
```

Comment:
Does the user have to refresh the page to "try again?" If so,

s/If the conditions have a status of `Unknown` or `False`, wait a few moments and then try again./If the conditions have a status of `Unknown` or `False`, wait a few moments to refresh the page./

OR 

s/^/If the conditions have a status of `Unknown` or `False`, wait a few moments and check again./